src/main/java/ru/yandex/qatools/embed/postgresql/PackagePaths.java

```diff
@@ -140,6 +140,18 @@ public String getPath(Distribution distribution) {
                 throw new IllegalArgumentException("Unknown BitSize " + distribution.getBitsize());
         }
 
-        return "postgresql-" + sversion + "-" + splatform + bitsize + "-binaries" + "." + sarchiveType;
+        String path = "postgresql-" + sversion + "-" + splatform + bitsize + "-binaries" + "." + sarchiveType;
+        switch (path) {
```

Comment:
It seems that it is better not to replace implicitly versions, and to write me Warning - suddenly the user wants to test this version

Reply:
@BorzdeG
https://github.com/julianladisch/postgresql-embedded/blob/eba601cc7d6602afea6511c170d1445c59d866cb/src/main/java/ru/yandex/qatools/embed/postgresql/distribution/Version.java defines
PRODUCTION(V10_1)
V10_1("10.1-1")
Have a look at the download page: https://www.enterprisedb.com/download-postgresql-binaries
10.1-1 is the correct sversion for Linux and Mac and Windows 32, but Windows 64 requires 10.1-2.
Trying to download https://get.enterprisedb.com/postgresql/postgresql-10.1-1-windows-x64-binaries.zip results in 403 Access Denied.
The same issue for V9_6(V9_6_6), V9_6_6("9.6.6-1") and https://get.enterprisedb.com/postgresql/postgresql-9.6.6-1-windows-x64-binaries.zip
A user that configures 10.1-1 or 9.6.6-1^most likely want -2 for Windows 64.
Removing lines 144-154 will fail the unit test that checks the Windows 64 download.
@BorzdeG If you have a better solution please create a pull request.